scripts/ceph-windows/run_tests

```diff
@@ -82,7 +82,7 @@ function collect_artifacts() {
     ssh_exec /wnbd/wnbd-client.exe version
     ssh_exec curl.exe --retry-max-time 30 --retry 10 -L -o /workspace/collect-event-logs.ps1 $COLLECT_EVENT_LOGS_SCRIPT_URL
     SSH_TIMEOUT=30m ssh_exec powershell.exe /workspace/collect-event-logs.ps1 -LogDirectory /workspace/eventlogs
-    scp_download /workspace/eventlogs $WORKSPACE/artifacts/client/eventlogs
+    SSH_TIMEOUT=10m scp_download /workspace/eventlogs $WORKSPACE/artifacts/client/eventlogs
```

Comment:
Are Windows event logs significantly bigger than other logs collected in this function?

It would be nice to:
- extend `scp_upload()` and `scp_download()` the same way as `ssh_exec()` was extended in https://github.com/ceph/ceph-build/pull/2116
- confirm that we are indeed hitting a timeout before proceeding with this PR

Reply:
> Are Windows event logs significantly bigger than other logs collected in this function?

The logs may get larger if there's an error that gets logged repeatedly. I saw that the log collection step failed only when there were failing tests.

> extend scp_upload() and scp_download() the same way as ssh_exec() was extended in ceph-windows: Re-enable Windows event logs collection #2116

Good point, thanks.

> Rather than singling out some call sites, we might want to increase the default timeout in scp_upload() and scp_download(). Currently it's the same as in ssh_exec(), which doesn't seem optimal -- uploading or downloading something is expected to take longer than executing a random command.

Makes sense, I've increased these timeouts to 10m.

Reply:
> Makes sense, I've increased these timeouts to 10m.

Then `SSH_TIMEOUT=10m` on this line isn't needed anymore?  And similarly there are two instances of `SSH_TIMEOUT=5m` specified for `scp_upload()` in other places -- probably should get rid of those as well.

Reply:
I was just being cautious in case the default timeouts get decreased but that probably won't be the case. I'm dropping these right away.